vm/systemSmartContracts/auction.go

```diff
@@ -523,6 +582,7 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 
 	if !isNumArgsCorrectToStake(args.Arguments) {
 		log.Debug("incorrect number of arguments to call stake", "numArgs", args.Arguments)
+		s.eei.AddReturnMessage("incorrect number of arguments to call stake")
```

Comment:
can we provide expectrd and got pattern here?

Reply:
here it think it is a little bit complicate to display how many arguments are expected